build/CMakeLists.txt

```diff
@@ -503,13 +503,13 @@ ExternalProject_Add(cnmem
 #
 ExternalProject_Add(aws-sdk-cpp
   PREFIX aws-sdk-cpp
-  GIT_REPOSITORY "https://github.com/aws/aws-sdk-cpp.git"
-  GIT_TAG "1.7.129"
+  GIT_REPOSITORY "https://github.com/outtoin/aws-sdk-cpp.git"
+  GIT_TAG "origin/feature/log-stsprofile"
```

Comment:
We usually want to stick to a tagged release and not a specific feature branch. If there is no release yet that contains this feature will the next release support it?

Reply:
I submit PR to aws-sdk-cpp repository, but they doesn't review it... :(
After they merge my PR, new release contains this feature, I'll fix it.